Proposed product descriptions/Organization Landing Page.md

```diff
@@ -0,0 +1,39 @@
+## Organization Landing Page
+
+### Summary
+This product is the creation of a landing page for organizations - these are groups of users that share credit for group contributions to KBase.
+
+### Story/Description
+In order to attract third party contributions, both data and tools, we need prominent branding of organizations so that data and tool contributions can be properly credited to an institution.  This is part of the larger problem of enabling groups in KBase, but much more specific, and may not necessarily imply sharing privileges.
+
+A User should be able to create an Organization, serve as an Organization admin, and allow users to be associated to the organization.  A landing page for each group should be available that allows a logo, description, links to be attached.  Tools and public workspaces can then be attached to the organization.  The organization page should display these contributions with usage statistics and organization team members.  Tools and workspaces in KBase should also be branded and linked to the organization page appropriately.
+
+### User stories
+As a user, I want to be able to create an organization and own it.
+
+As a user, I want to request to join (or leave) an organization.
+
+As an org admin, I want to invite and admit users into my organization.
+
+As an org admin, I want to remove users from my organization.
+
+As an org admin, I want to add branding to my organization's page. This would include images, links, and text about my organization.
+
+As a user, I want to see organization information on that landing page, including the branding information, which apps are owned by that organization, and which public Narratives (and data) are owned by that organization.
+
+
```

Comment:
What about searching users to find users to add to your organization? Listing users who belong to a particular organization (will that be restricted to people who are themselves in the organization?). Searching for an organization?
Can users add themselves to an organization, or do they need to request that and have the request handled by an org admin? If the latter, how will org admins be notified about such requests?